### ISSUE TITLE
leaflet: cant find bounds (<Typ3 Version 9.5.7)

### ISSUE THREAD

**Issue by LPGGNecromant** [Treedent/rt_simpleosm#7]
leaflet cant find bounds after Typo3 Version 9.5.7.

```
leaflet.js:5 Uncaught Error: Bounds are not valid.
    at e.fitBounds (leaflet.js:5)
    at XY:353
```

Workaround currently for me was, load Typo3 V. 9.5.7, save the Map-Item, load back Typo3 V. 9.5.14.

Example (what Typo3 does with it): https://codepen.io/Necromant/pen/WNvYzXP

**Comment by Cyasi** [comment]
Same issue in 9.5.15.
Fixed with this patch from bergischweb: https://github.com/Treedent/rt_simpleosm/issues/4#issuecomment-536934201

**Comment by Treedent** [comment]
Fixed with version 4.1.2
